### PR TITLE
Get cart handler.

### PR DESCRIPTION
**Pull request by sridhar02**
#8

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -280,6 +279,69 @@ func postToCartHandler(c *gin.Context, db *sql.DB) {
...
+	var userId int
+	row := db.QueryRow(`SELECT user_id FROM logins WHERE secret=$1`, secret)
+	err := row.Scan(&userId)
+	if err != nil {
+		fmt.Println(err)
+		c.AbortWithStatus(http.StatusInternalServerError)
+		return
+	}
+
+	rows, err := db.Query(`SELECT product_id FROM cart where user_id =  $1`, userId)
```
Comment (vramana) [inline main.go]: Do join on products table instead.

File: main.go [hunk 2]
```diff
@@ -280,6 +279,69 @@ func postToCartHandler(c *gin.Context, db *sql.DB) {
 	c.Status(http.StatusCreated)
 }
 
+func getCartHandler(c *gin.Context, db *sql.DB) {
+
+	value := c.GetHeader("Authorization")
+	secret := strings.TrimPrefix(value, "Bearer ")
+
+	var userId int
+	row := db.QueryRow(`SELECT user_id FROM logins WHERE secret=$1`, secret)
```
Comment (vramana) [inline main.go]: Mov this into a function.

File: main.go (hunk 1 above)
Comment (sridhar02) [reply main.go]: done

File: main.go (hunk 2 above)
Comment (sridhar02) [reply main.go]: done

**Pull request merged**